src/main/java/com/bugsnag/android/Configuration.java

```diff
@@ -386,4 +394,11 @@ protected boolean inProject(String className) {
 
         return false;
     }
+
+    private void notifyObservers(NotifyType type) {
+        if (Bugsnag.client != null) {
```

Comment:
This assumes that people are using the static `Bugsnag` class, when they are possibly using a `Client` instance instead. Each `Client` instance will have their own instance of `Configuration`.